### ISSUE TITLE
Progress not saving 

### ISSUE THREAD

**Issue by Phil-regicide** [HammerfallSupport/Warhammer40k-Regicide-Support#2]
Hi I have logged in on my profile on my iPad after setting up on my phone (android) and my campaign progress moved across as well. 
now whenever I complete a mission my progress doesn’t save.

**Comment by xSaimuraix** [comment]
Yes I’m getting this issue also. I played all of campaign 1 and half of 2 then it stopped saving my progress. So I rebooted and signed in this time, of course no progress was saved so I started again played several missions and quit, watching it say “saving to online profile” each time. When I came back I had to start mission 1 again. If I look in the list I can see that 2,3&4 aren’t completed yet 5-9 are. If I do them all again, 10 doesn’t save and I have to go back to 1 again anytime I close the app.
